src/components/App.js

```diff
@@ -76,43 +79,16 @@ function App() {
         return newBag
       })
   }
+ 
+  const addToShoppingBag = (item) => {
+    setShoppingBag([...shoppingBag, item]);
+  }
     
   const getItemsForDisplay = () => {
-    const productsForDisplay = [];
-    
-    const productNames = new Set (items.map(product => product.name))
-    
-    productNames.forEach(productName => {
-      
-      const filteredProducts = items
-        .filter(product => product.name === productName)
-      
-      const filteredProduct = filteredProducts.reduce((acc, curr) => {
-        acc.id = curr.name;
-        acc.name = curr.name;
-        acc.price = curr.price;
-        acc.category = curr.category
-        acc.image = curr.image
-        acc.description = curr.description
-        if (!acc.colorOptions.includes(curr.color)) {
-          acc.colorOptions.push(curr.color)
-        }
-        if (!acc.sizeOptions.includes(curr.size)) {
-          acc.sizeOptions.push(curr.size)
-        }
-        return acc;
-      }, {
-        colorOptions: [],
-        sizeOptions: []
-      })
-      productsForDisplay.push(filteredProduct)
-    }) 
-    return productsForDisplay
+    return cleanFetchedData(items);
```

Comment:
We could probably get rid of this getItemsForDisplay function altogether and run the cleanFetchedData  function on line 99 instead yeah?

Reply:
Yes. Will do